.gitignore

```diff
@@ -12,3 +12,4 @@ target
 *.code-workspace
 .bloop
 .metals
+dumps
```

Comment:
when I test in local ,this folder generate, I think adding it to ignore  file would be better